Build instructions broken

[@t184256 reports](https://gitlab.com/redhat-crypto/fedora-crypto-policies/-/merge_requests/122#note_1157998580):

> I'm very sorry, but I don't know how to test it all. AFAIU, sequoia proper isn't looking at crypto-policy, as support for it is added on rpm-sequoia level (can't say I understand why). And I can't build rpm with rpm-sequoia, as the instructions in README seem outdated and I'm stuck at RPMSEQUOIA_LDFLAGS:INTERNAL=-L/tmp/rpm/rpm-sequoia/target/release;-lrpm_sequoia, yet that -L doesn't make it to the invocation and I'm stuck with /usr/bin/ld: cannot find -lrpm_sequoia.
> 
> Could you update the instructions? Or, better yet, provide some command to verify the config so that I can use it in CI and during build tests?

--- a/README.md
+++ b/README.md
@@ -47,6 +47,8 @@ $ cd /tmp/rpm
 $ git clone git@github.com:rpm-software-management/rpm.git
 Cloning into 'rpm'...
 done.
+$ git checkout -b rpm-4.18.0-release
+Switched to a new branch 'rpm-4.18.0-release'
 $ cd rpm
 $ autoreconf -fis
 ...
@@ -58,6 +60,13 @@ $ ../configure --with-crypto=sequoia
 $ make
 $ make check
 ```
+
+Note: this builds version 4.18 of `rpm`, which is the current stable
+release of `rpm`.  The current development branch of `rpm` has
+switched to using `cmake` instead of `autoconf`.  Please refer to
+[rpm's `INSTALL`] file for how to build `master`.
+
+  [rpm's `INSTALL`]: https://github.com/rpm-software-management/rpm/blob/master/INSTALL
 
 To use a different cryptographic backend, you need to disable the
 default backend, and select your preferred backend.  For instance, to

